push_notifications/models.py

```diff
@@ -137,7 +137,7 @@ def send_message(self, message, certfile=None, **kwargs):
 				)
 				r = apns_send_bulk_message(
 					registration_ids=reg_ids, alert=message, application_id=app_id,
-					certfile=certfile, **kwargs
+					creds=creds, **kwargs
```

Comment:
is there any effect of this change on migration?

Reply:
FWIW I migrated without issue, with a relatively simple setup.